mozci/queries/test_task_overhead.query

```diff
@@ -0,0 +1,11 @@
+from: unittest
+format: list
+groupby: task.id
+limit: 20000
```

Comment:
Do we need such a high limit? If so, we might want to use `destination: url` like https://github.com/mozilla/mozci/blob/737c4cc0810bd745d13c90c511d18afff6baee20/mozci/queries/push_revisions.query.

Reply:
No we do not - this was just carried over from the working query. It should return only 1 row.